packages/api/tests/endpoints/granules/test-bulk-delete.js

```diff
@@ -16,6 +16,7 @@ const {
   setAuthorizedOAuthUsers,
 } = require('../../../lib/testUtils');
 const AccessToken = require('../../../models/access-tokens');
+const { granuleIds } = require('../../helpers/granulesList');
```

Comment:
Can we dynamically create the granuleId list?
Also the granuleId can be very long, e.g. SWOT_L2_HR_Raster_100m_UTM12R_N_x_x_x_007_162_051F_20220809T224919_20220809T224939_Dx0000_01

Reply:
https://github.com/nasa/cumulus/pull/3122/commits/a060b7c2f7d55d16e3de0a6a224ae3d9747295cd

Reply:
https://github.com/nasa/cumulus/pull/3122/commits/31e1e92add3cb360f937f62e0121821f3eeb50f7